Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM funkyfuture/nest-of-serpents
```

Comment:
just add a note that @funkyfuture might behave nasty at some point and runnng the tests with that script is on everyone's own risk?